azure-kusto-data/azure/kusto/data/security.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, kcsb):
         else:
             self._authentication_method = AuthenticationMethod.aad_device_login
             self._client_id = "db662dc1-0cfe-4e1c-a843-19a68e65be58"
-            create_adal_context = False
+            create_adal_context = True
```

Comment:
are you sure ?

Reply:
Yes, the previous change disabled the device authentication by mistake.